tests/unit/memory_storage/resource_clients/test_key_value_store.py

```diff
@@ -1,39 +1,41 @@
 import asyncio
 import os
+from pathlib import Path
 
 import pytest
 
-from apify.memory_storage import MemoryStorage
-from apify.memory_storage.resource_clients import KeyValueStoreClient
+from apify._memory_storage import MemoryStorageClient
+from apify._memory_storage.resource_clients import KeyValueStoreClient
 
 
 @pytest.fixture
-async def key_value_store_client(memory_storage: MemoryStorage) -> KeyValueStoreClient:
-    key_value_stores_client = memory_storage.key_value_stores()
+async def key_value_store_client() -> KeyValueStoreClient:
+    memory_storage_client = MemoryStorageClient()
```

Comment:
Why create a new instance here? I see it injected in other tests. e.g. dataset

Reply:
Right, I was just playing with something but then forgot to revert it here, I will change it back to the inject